Lib/test/test_signal.py

```diff
@@ -124,6 +124,8 @@ def __repr__(self):
         self.assertEqual(0, argument.repr_count)
 
     def test_strsignal(self):
+        if sys.platform.startswith("netbsd"):
```

Comment:
Maybe you should add a NEWS to account for your modifications, That's i think

Reply:
tests usually don't need NEWS, and in part, this is invisible to users